itest/tests/exportResults.test.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import fsExtra from "fs-extra"
+
+import dialog from "spectron-fake-dialog"
+import path from "path"
+
+import {
+  click,
+  newAppInstance,
+  pcapIngestSample,
+  startApp,
+  startSearch,
+  writeSearch
+} from "../lib/app"
+import {locator} from "../../src/js/test/integration"
+import {retryUntil} from "../lib/control"
+
+describe("test exporting results", () => {
+  let app
+  let testIdx = 0
+
+  beforeAll(async () => {
+    app = newAppInstance(path.basename(__filename), ++testIdx)
+    dialog.apply(app)
+    await startApp(app)
+    await pcapIngestSample(app)
+    await fsExtra.ensureDir("tmp")
+  })
+
+  afterAll(async () => {
+    if (app && app.isRunning()) await app.stop()
+    await fsExtra.remove("tmp")
+  })
+
+  test("clicking the button", async () => {
+    let filePath = path.normalize("tmp/test-export-results.zng")
+    dialog.mock([
+      {
+        method: "showSaveDialog",
+        value: {
+          canceled: false,
+          filePath
+        }
+      }
+    ])
+    await writeSearch(app, "")
```

Comment:
It would be good to have a few tests that actually perform zql searches and verify the results come from that search, and not something else or the whole space. There are a few ways to do this and some considerations. First, the zql should be deterministic: you should apply some sort of `sort` so that the data always comes back in the same order. Second, the output zng file should be inspected. With pcaps, I did this using md5 checksums, after first locally verifying the pcap contained the correct output. Or you could do something more complex like call `zq`.